src/pages/_layout.svelte

```diff
@@ -66,11 +66,33 @@ $: menuItems = [
     hide: inAdminRole || !policyId,
   },
 ]
+const isCustomerOnOwnPolicy = (policyId: string) => policyId === $selectedPolicyId
 
-const goToPolicyAsCustomer = (event: CustomEvent) => $goto(routes.policyHome(event.detail))
-const goToAdminHome = () => $goto(routes.ADMIN_HOME)
+const gotoPath = (policyId: string, claimOrItemIdObj = {}) =>
+  $goto($url($route.path, { policyId, ...claimOrItemIdObj }))
```

Comment:
`$goto()` will handle the same parameters as `$url()`, so you can remove the call to `$url()` here:
```suggestion
  $goto($route.path, { policyId, ...claimOrItemIdObj })
```